@here/harp-mapview/lib/Utils.ts

```diff
@@ -410,6 +418,257 @@ export namespace MapViewUtils {
         return result;
     }
 
+    /**
+     * @hidden
```

Comment:
AFAIK `@hidden` is only for typedoc.
Since we want to use api-extractor in the future I suggest to use to these tags:
https://api-extractor.com/pages/tsdoc/doc_comment_syntax/#release-tags

I can also do this as part of my (never ending) API task

Reply:
Added both @hidden and @internal.